concordium-node/src/configuration.rs

```diff
@@ -467,7 +467,9 @@ pub struct ConnectionConfig {
     #[structopt(
         long = "no-clear-bans",
         help = "Do not clear the ban database on start.",
-        env = "CONCORDIUM_NODE_CONNECTION_NO_CLEAR_BANS"
+        env = "CONCORDIUM_NODE_CONNECTION_NO_CLEAR_BANS",
+        parse(try_from_str),
```

Comment:
Please don't do this. This makes the behaviour of this one flag different from all the other boolean flags.

Reply:
Instead you should change the environment variable to `CONCORDIUM_NODE_CLEAR_BANS` and document that in the changelog